testdata/baselines/reference/submodule/conformance/privateNameMethodClassExpression.errors.txt

```diff
@@ -13,9 +13,9 @@ privateNameMethodClassExpression.ts(10,17): error TS18013: Property '#field' is
     console.log(C.getInstance().getField());
     C.getInstance().#method; // Error
                     ~~~~~~~
-!!! error TS18013: Property '#method' is not accessible outside class '(Missing)' because it has a private identifier.
+!!! error TS18013: Property '#method' is not accessible outside class 'C' because it has a private identifier.
     C.getInstance().#field; // Error
                     ~~~~~~
-!!! error TS18013: Property '#field' is not accessible outside class '(Missing)' because it has a private identifier.
+!!! error TS18013: Property '#field' is not accessible outside class 'C' because it has a private identifier.

```

Comment:
Strada would print `(anonymous)` here but this has an inferrable~ name from the declaration and I think it's better to use it. Using it is not a new thing to do.